appinterface/projection/crossfire/httpclient.go

```diff
@@ -23,22 +24,20 @@ func NewHTTPClient(participantsListURL string) *HTTPClient {
 
 	return &HTTPClient{
 		httpClient,
-		participantsListURL,
+		strings.TrimSuffix(participantsListURL, "/"),
 	}
 }
 
 func (client *HTTPClient) Participants() (*[]Participant, error) {
-	rawRespBody, err := client.request(client.url)
+	rawRespBody, err := client.request("participants.json")
```

Comment:
May I know why we hardcoded a `participants.json` here? and also the current participants URL in config is still full URL right?

Reply:
Yes, THanks for asking this. Even I didn't want to do this, but due to the `ghttp` ginkgo unit testing limitation it had to be done. I couldn't figure out a way to keep it like previous and get the unit test to pass.
Maybe if we find out some way then we can go back to original.

Reply:
Update here:
Managed to run and add test case using ghttp now. Thanks. Have pushed again